app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -210,7 +229,11 @@ public function getWebsite($websiteId = null)
     }
 
     /**
-     * {@inheritdoc}
+     * Get loaded websites
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc